library/build.gradle

```diff
@@ -17,14 +17,12 @@ ext {
 apply from: "${rootDir}/scripts/publish-module.gradle"
 
 android {
-    compileSdkVersion 30
+    compileSdkVersion 32
```

Comment:
This is only what it compiles against when writing code, not what the minimum supported version is. We support all the way down to version 21 (Android 5.0) which is from about 2014. Also, if we were using any features that only happened after 30, the IDE would warn us that we support more users than that feature allows.

I also went ahead and did this because Google is starting to restrict these versions and require devs to upgrade them. If one of our customers is trying to get compliant with Google by upgrading our versions, our library could potentially hold them back if these versions weren't kept up w/ Google's constraints.